src/app/subnets/views/VLANDetails/DHCPStatus/DHCPStatus.tsx

```diff
@@ -64,7 +64,7 @@ const DHCPStatus = ({ id, openForm }: Props): JSX.Element | null => {
     vlanSelectors.getById(state, id)
   );
   const vlanSubnets = useSelector((state: RootState) =>
-    subnetSelectors.getByVLAN(state, id)
+    subnetSelectors.getByIds(state, vlan ? vlan.subnet_ids : [])
```

Comment:
You stated in the PR description:
> "Use list of IDs to fetch subnets (where it makes sense to do so)"

Just wanted to clarify in this implementation we're not actually fetching subnets by ids, only using a different selector for retrieving subnets that were already fetched from the redux store.

Reply:
I don't think fetching subnets by ids is actually possible with `subnet.list` WebSocket API method, which currently simply returns all subnets.

Reply:
Oh damn... So we'd just need to fetch the full list of subnets anyway in order to use this?